src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/colors.scss

```diff
@@ -8,13 +8,12 @@ $alto: #ddd;
 $black: #000;
 $blue: #00f;
 $blueZodiac: #112a46;
```

Comment:
last time i checked, this file contained a lot of similar blues. maybe can we remove some of them now?

Reply:
maybe some of them are used in a single place only?